src/SonarQube.Client/Models/ServerSentEvents/ClientContract/IIssueChangedServerEvent.cs

```diff
@@ -53,6 +54,19 @@ public IssueChangedServerEvent(string projectKey, bool isResolved, BranchAndIssu
 
         [JsonProperty("issues")]
         public IBranchAndIssueKey[] BranchAndIssueKeys { get; }
+
+        // Display-friendly override for logging/debugging
+        public override string ToString()
+        {
+            var sb = new StringBuilder();
+            sb.AppendLine($"ProjectKey: {ProjectKey}, IsResolved: {IsResolved}, Issue count: " + BranchAndIssueKeys.Length);
+            foreach (var item in BranchAndIssueKeys)
+            {
+                sb.AppendLine($"\tBranch: {item.BranchName}, IssueKey: {item.IssueKey}");
+            }
+
+            return sb.ToString();
```

Comment:
Overriding `ToString` so that
```
logger.LogVerbose(Resources.Suppression_IssueChangedEventReceived, issueServerEvent);
```
... prints something useful.

This `ToString` method uses a StringBuilder i.e. creates objects and calls other functions to generate the text.

However,  the way the `LogVerbose` method is written means it will only be called when we are actually in verbose logging mode - it won't be called in "normal" logging mode.